lang/java/java.talon

```diff
@@ -0,0 +1,87 @@
+mode: user.java
+mode: command
+and code.language: java
+
+-
+tag(): user.code_operators
+tag(): user.code_comment
+tag(): user.code_generic
+
+action(user.code_operator_indirection): ""
+action(user.code_operator_address_of): ""
+action(user.code_operator_structure_deference): ""
+action(user.code_operator_lambda): ""
```

Comment:
for things that aren't supported. I'd recommend using `skip()`

Reply:
Done.